app-react/src/App.tsx

```diff
@@ -1,26 +1,109 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { useEffect, useState } from 'react';
+import { Button, Stack, TextField, Typography } from '@mui/material';
+import SendIcon from '@mui/icons-material/Send';
+
+import { createTransaction, getServerStatus, getTransactions } from './api/requests';
+import { Transaction } from '../../app-express/src/api/types';
+import { TransactionView } from './components/TransactionView';
+
```

Comment:
As I felt as I should keep this app as simple/vanilla as possible, I had to have both "views" in the same file as to enable side effects for when creating a new transaction, it should also fetch the new transaction histories.